src/app/(app)/(home)/search-filters/categories.tsx

```
@@ -0,0 +1,23 @@
+import { CategoryDropdown } from "~/app/(app)/(home)/search-filters/category-dropdown";
+
+interface CategoriesProps {
+	data: any;
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)